project.clj

```diff
@@ -1,3 +1,5 @@
+(def kafka-client-version "2.2.0")
```

Comment:
this is not used anywhere, can we remove it?

Reply:
done